letsdrill.rb

```diff
@@ -1,24 +1,56 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+
+  if integer.between?(90,100)
+    return "A"
+  elsif integer.between?(75, 90)
+    return "B"
+  elsif integer.between?(60, 74)
+    return "C"
+  end
+
 
 end
 
 def shortest_string(array)
 
-  #Put your code here!
+  if array == []
+    return nil
+
+  elsif array.length == 1
+    return array
+
+  elsif array.length > 1
+    temp0 = array[0]
+    temp1 = array[1]
+
+    if temp0.length < temp1.length
+      keep = temp0
+    else
+      keep = temp1
+    end
+
+    for index in 2 ... array.size
```

Comment:
Look through the array documentation for an iterator that might be appropriate here for iterating through your array and selecting one of them.